events/dvg_england.txt

```diff
@@ -0,0 +1,786 @@
+﻿namespace = dvg_england

```

Comment:
Left and right side do not have matching types. Left has type(s) 'country', right has type(s) 'value' at ' file: events/dvg_england.txt line: 239'